### DISCUSSION TITLE
I am getting this error when compiling livecharts2 Error MSB3644

### DISCUSSION THREAD

**Discussion by humbertomoli99** [Live-Charts/LiveCharts2#188] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/22565720/133379488-bee0790f-95c2-4b89-8117-f78e9d8d3f72.png)

Error MSB3644 No se encuentran los ensamblados de referencia para .NETFramework, Version = v4.6.2. Para resolver este problema, instale el Paquete de desarrollador (SDK o Paquete de compatibilidad) de esta versión de .NET Framework o redestine la aplicación. Para descargar los paquetes de desarrollador de .NET Framework, vaya a https://aka.ms/msbuild/developerpacks LiveChartsCore.SkiaSharpView.Avalonia C: \ Archivos de programa (x86) \ Microsoft Visual Studio \ 2019 \ Community \ MSBuild \ Current 
\ Bin \ Microsoft.Common.CurrentVersion.targets

**Comment by beto-rodriguez** [comment, marked as answer]
Install the 4.6.2 sdk

https://dotnet.microsoft.com/download/visual-studio-sdks

**Reply by humbertomoli99** [reply]
Thank you very much, it helped me
